scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -246,13 +246,13 @@ it is useful when having multiple lanes with the same name, but with different r
 
   async report([laneName, alias]: [string, string, string]): Promise<string> {
     const { laneId } = await this.lanes.aliasLane(laneName, alias);
-    return `successfully added the alias ${chalk.bold(alias)} to the lane ${chalk.bold(laneId.toString())}`;
+    return `successfully added the alias ${chalk.bold(alias)} for lane ${chalk.bold(laneId.toString())}`;
   }
 }
 
 export class LaneChangeScopeCmd implements Command {
   name = 'change-scope <lane-name> <remote-scope-name>';
-  description = `changes the remote scope of a lane`;
+  description = `changes the remote scope of a lane. NOTE: available only before the lane is exported to the remote`;
```

Comment:
Please move this note to the `extendedDescription`.

Reply:
done